### DISCUSSION TITLE
z.enum() with createUniqueFieldSchema()

### DISCUSSION THREAD

**Discussion by alxcrt** [iway1/react-ts-form#9] [General]
<img width="573" alt="image" src="https://user-images.githubusercontent.com/20427840/210466666-79626e05-24f0-4ad0-ab07-f90503875485.png">

I want to create unique fields with enums. But I don't know how to pass custom values. Because in the current implementation only "placeholder" will show.

**Comment by iway1** [comment]
I've deprecated enums because they seem to be confusing and be more of a footgun than anything. I'd recommend using other field types since they don't have any of these issues.

Radio field can be implemented with `z.string()`, checkbox with `z.boolean()`, etc.
